src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesService.kt

```diff
@@ -75,25 +75,25 @@ class CaseNotesService(
 
   override suspend fun retryCreateMapping(message: String) = createMapping(message.fromJson())
 
-  suspend fun amendCaseNote(caseNoteEvent: CaseNoteEvent) {
+  suspend fun updateCaseNote(caseNoteEvent: CaseNoteEvent) {
     val dpsCaseNoteId = caseNoteEvent.additionalInformation.caseNoteUuid
     val offenderNo = requireNotNull(caseNoteEvent.personReference.findNomsNumber())
     val telemetryMap = mutableMapOf(
       "dpsCaseNoteId" to dpsCaseNoteId,
       "offenderNo" to offenderNo,
     )
 
-    if (caseNoteEvent.wasAmendInDPS()) {
+    if (caseNoteEvent.wasAmendedInDPS()) {
       runCatching {
         val mapping = mappingApiService.getOrNullByDpsId(dpsCaseNoteId)
           ?: throw IllegalStateException("Tried to amend an casenote that has never been created")
```

Comment:
Is the message right on this exception?

Reply:
Its a bit oddly expressed thats true, ill change it next time